app/assets/stylesheets/syntax.scss

```diff
@@ -13,7 +13,7 @@ div.inner-comment div.body div.highlight pre.highlight {
   }
 }
 
-$code-highlight-background-color: #29292e;
+$code-highlight-background-color: #08090a;
```

Comment:
For a future PR perhaps, but does it make sense to convert these to CSS properties like we've done with many other SASS vars?

Reply:
Probably but not necessarily per theme because syntax highlighter likely won't change no matter what theme. So not global `:root` variables but yea - we can convert them to be CSS variables indeed.